ingress/controllers/nginx-third-party/main.go

```diff
@@ -62,14 +62,24 @@ func main() {
 		glog.Fatalf("Please specify --default-backend")
 	}
 
+	glog.Info("Checking if DNS is working")
+	ip, err := checkDNS(*defaultSvc)
```

Comment:
@bprashanth I will try to remove this requirement in another PR creating a lookup table (dns name, svc ip)